data/src/test/java/com/example/data/mappers/WeatherDailyDataMapperTest.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.data.mappers
+
+import com.example.data.modelsApi.currentWeather.CurrentWeatherApiModel
+import com.example.data.modelsApi.currentWeather.Data
+import com.example.data.modelsApi.currentWeather.Weather
+import com.example.data.modelsApi.multiDaysWeather.DailyData
+import com.example.data.modelsApi.multiDaysWeather.DailyWeather
+import com.example.data.modelsApi.multiDaysWeather.DailyWeatherApi
+import com.example.domain.models.WeatherData
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+internal class WeatherDailyDataMapperTest {
+
+    private val weatherDataMapper = WeatherDataMapper()
+    private val iconUrl = "https://www.weatherbit.io/static/img/icons/"
```

Comment:
Use constant ICON_URL that is used in WeatherDataMapper class. To prevent code duplicating.

Reply:
Fixed